client/me/purchases/remove-purchase/index.jsx

```diff
@@ -251,10 +305,14 @@ class RemovePurchase extends Component {
 							//{ components: { domain: <em>{ getIncludedDomain( purchase ) }</em> } }
 						} )
 					}{ ' ' }
-					{ translate(
-						'You will not be able to reuse it again without purchasing a new subscription.',
-						{ comment: "'it' refers to a product purchased by a user" }
-					) }
+					{ isDomainRegistration &&
+						translate(
+							'You will not be able to reuse it again without starting a new subscription.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.